### ISSUE TITLE
Issue with Accessing Raw Data Link for Human Heart scRNA-seq Atlas

### ISSUE THREAD

**Issue by aqlkzf** [YangLabHKUST/SpatialScope#12]
Hello,

I was going through the [Human Heart tutorial](https://spatialscope-tutorial.readthedocs.io/en/latest/notebooks/Human-Heart.html) and encountered an issue when trying to access the raw dataset for the human heart scRNA-seq atlas. The link provided in the section "We will use a human heart snRNA-seq atlas as reference dataset, the raw dataset in h5ad format (global_raw.h5ad) is available in [here](www.heartcellatlas.org)" appears to be broken or unavailable.

Could you please update the method for downloading the raw data or fix the existing link to ensure accessibility?

Thank you for your assistance.

Best regards,
Jishuai

<img width="1591" alt="image" src="https://github.com/YangLabHKUST/SpatialScope/assets/67051837/95fc6381-8764-4a4c-a101-10808a122bf3">

<img width="1198" alt="image" src="https://github.com/YangLabHKUST/SpatialScope/assets/67051837/87a43acd-bc10-44db-968f-ca4b054281cf">

**Comment by aqlkzf** [comment]
Hi,

Following up on my earlier issue with the Human Heart scRNA-seq atlas dataset, I downloaded the `global_raw.h5ad` file from [Heart Cell Atlas](https://www.heartcellatlas.org/) as per Litviňuková et al., Nature 2020. However, using Scanpy's `sc.pp.filter_cells(ad_sc, min_counts=500)` resulted in an empty dataset `(0, 32732)`, suggesting all cells were filtered out.

Also, the dataset lacked a `ad_sc.obs['cell_source']` column. I improvised by merging `facility` and `cell_or_nuclei` columns:
```python
ad_sc.obs['cell_source'] = str(ad_sc.obs['facility']) + '-' + str(ad_sc.obs['cell_or_nuclei'])

**Comment by aqlkzf** [comment]
I've solved the problem according to the description in the original paper. For those looking to access the raw dataset, the correct download link is: https://cellgeni.cog.sanger.ac.uk/heartcellatlas/v2/Global_raw.h5ad

Additionally, I noticed a small adjustment needed for the dataset to work smoothly with the preprocessing steps outlined in the Human Heart tutorial (`docs/source/notebooks/Human-Heart.ipynb`):

1. **Adding `cell_source` Column**: To align with the tutorial's preprocessing steps, you'll need to add an `obs` column named `cell_source`. This can be done with the following line of code:

```python
ad_sc.obs['cell_source'] = ad_sc.obs['facility'].astype(str) + '-' + ad_sc.obs['cell_or_nuclei'].astype(str)
```


2. **Correction in Plotting Function**: In the preprocessing section (`## Preprocessing scRNA-ref`), the correct parameter for the last plotting function should be `color="cell_state"`, not `color="cell_states"`.

I hope these updates help others in their exploration of this fascinating tool.
